### DISCUSSION TITLE
what is the procedure to translate the subtitles ?

### DISCUSSION THREAD

**Discussion by neozbr** [abus-aikorea/voice-pro#28] [General]
I would like to understand what version of whisper or what procedure  is done to translate the subtitles, because its really good, I've tried many whisper  versions and appz and none translate correctly like this..would you know what the procedure is used or what whisper version is used for this or what is the procedure for the subtitle translation if you can tell me please,thanks.

**Comment by abus-aikorea** [comment]
Thank you for highly appreciating ABUS's voice-pro. We have been working for a long time to create subtitle, translation, and dubbing solutions for YouTube creators, online course video producers, and broadcast content creators. We have tested the transcription capabilities and accuracy of all whisper versions, considering input and output for effective translation. However, our system still has many errors. Therefore, we have modified the program to allow for direct review and editing of the transcribed and translated text. We hope this feature of voice-pro will be helpful to you, and we regret that we cannot share our know-how in detail. If you have any further questions, please feel free to let us know. Thank you.
